Remove the native build and PicoCLI to allow support for multiple platforms (#56)
strimzi/drain-cleaner: development-docs/DEVELOPING.md

mvn compile quarkus:dev
```

### Building the Java code with Maven

You can build the project using Maven.

> Once you have Java and Maven installed, you can run to build the project  

Reviewer: ```suggestion
With Java and Maven installed, you can run Maven to build the project. 
```

```shell script
mvn package
```

> ### Creating a native executable using Make

Reviewer 2: If we're removing native image support I guess we need to remove this heading too?
